package.json

```diff
@@ -60,11 +60,10 @@
   "dependencies": {
     "@tensorflow/tfjs": "^1.5.1",
```

Comment:
We need to keep TFJS out of the dependencies!

Reply:
This is addressed in #98 :)